erts/emulator/beam/beam_bif_load.c

```diff
@@ -1127,11 +1127,10 @@ check_process_code(Process* rp, Module* modp, int *redsp, int fcalls)
     /*
      * Check if current instruction or continuation pointer points into module.
```

Comment:
```suggestion
     * Check if current instruction points into module.
```

Or alternatively remove the whole comment? :)